sync.sh

```diff
@@ -145,9 +163,15 @@ ignore = Name *___jb_tmp___*
 # Additional user configuration
 $SYNC_EXTRA_UNISON_PROFILE_OPTS
 
-" > /root/.unison/default.prf
+" > ${HOME}/.unison/default.prf
+
+# Fix permissions
+log_heading "Override folder permissions."
+
+chown -R ${UNISON_USER}:${UNISON_GROUP} $SYNC_SOURCE
+chown -R ${UNISON_USER}:${UNISON_GROUP} $SYNC_DESTINATION
```

Comment:
Yes, when $SYNC_SOURCE or $SYNC_DESTINATION are host mounted volumes.
This has the potential to be a very slow operation and this behavior can impact existing bg-sync users. I'll rollback this change.